src/runtime/auto-prompt-manager.ts

```diff
@@ -930,7 +986,10 @@ export class AutoPromptManager {
     }
 
     // ** Frequency Capping Impressions **
-    if (this.frequencyCappingLocalStorageEnabled_) {
+    if (
+      this.frequencyCappingByDismissalsEnabled_ ||
```

Comment:
What's the ramp-up strategy for the new flag? 
- Are we going to turn down ongoing experiment and set up a new experiment with dismissal that goes up to 50%?
- Or are we going to keep the impression experiment and run a new experiment with dismissals that sits on top of it and goes up to 100%?

If the latter, wouldn't it be accurate to only refer to `frequencyCappingLocalStorageEnabled_` to keep it to 50%?

Reply:
Good point, keeping this check to only `frequencyCappingLocalStorageEnabled_` so that the new flag will be able to ramp to 100%.